res/values-tr/strings.xml

```diff
@@ -1168,7 +1168,7 @@ Bu, özel görünümlerde kullanılması *YASAK* olan şeylerin bir listesidir;
     <string name="PRIVATE">"ÖZEL"</string>
     <string name="Joined">"Katılım"</string>
     <string name="Requests">"İstekler"</string>
-    <string name="Invites">"Davetler"</string>
+    <string name="Invites">"Davetiyeler"</string>
```

Comment:
Davetiyeler =invitations
Davetler =İnvites
This word has already been translated correctly